plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -11,7 +11,12 @@
 from functools import partial
 
 from dbt.version import get_installed_version
-from dbt.config.profile import PROFILES_DIR
+
+try:
+    from dbt.flags import PROFILES_DIR # dbt >= 1.0
+except:
```

Comment:
```suggestion
except ImportError:
```
don't use bare except